modules/fts/pages/fts-architecture.adoc

```diff
@@ -0,0 +1,54 @@
+= FTS Architecture
+
+
+[abstract]
+
+
+The FTS Architecture is explained in the form of diagrams. Each diagram is co-located with its textual description.
+
+== Search Service Architecture
+
+The _Search_ Service provides extensive capabilities for natural-language querying.
+These include:
+
+* _Language-aware_ searching; allowing users to search for, say, the word `beauties`, and additionally obtain results for `beauty` and `beautiful`.
+* _Scoring_ of results, according to relevancy; allowing users to obtain result-sets that only contain documents awarded the highest scores.
+This keeps result-sets manageably small, even when the total number of documents returned is extremely large.
+* _Fast indexes_, which support a wide range of possible text-searches.
+
+The indexes that the Search Service creates and uses are entirely separate from and different to those of the Index Service.
+
+However, if the Index and Query Services are also deployed on the cluster, the facilities of the Search Service can be called on by means of the _N1QL_ language &#8212; which provides the principal interface to the Query and Index Services.
+See xref:learn:services-and-indexes/services/search-service.adoc#search_via_query[Accessing the Search Service with N1QL], below.
+
+The Search Service depends on the Data Service, as follows:
+
+[#search_service_architecture]
+image::services-and-indexes/services/searchServiceArchitecture2.png[,500,align=left]
+
+The shows the following:
+
+* The _Data Service_ uses the DCP protocol to stream data-mutations as batches, from the producer to multiple consumers (_vBuckets_), instantiated across the search nodes.
+
+* When a _search index_ is created by means of the _Search Service_, search-index data-handling for the vBuckets is divided equally among the established search-index partitions.
+
+For example, if the number of vBuckets is 120, and the number of search-index partitions chosen is 6, each search-index partition holds data for 20 vbuckets.
+
+All available Search Service nodes in the cluster are individually searchable.
+When one particular Search Service node is chosen for a search request, it assumes the role of _coordinator_; and is thereby responsible for applying the search request to the other Search Service nodes and for gathering and returning results.
+The following illustration depicts this _scatter-gather_ execution of a search request:
+
+[#search_operation]
+image::services-and-indexes/services/searchServiceOperation.png[,700,align=left]
+
+This illustration shows how:
+
+* The application makes a search request to a specific Search Service node (here, _Node 1_).
+This node assumes the role of coordinator.
+
+* The coordinator scatters the search request to all other search-index partitions (here, _Node 2_ and _Node 3_) in the cluster.
+
+* Once all the returned data is gathered, the coordinator applies filters as appropriate and returns the final results to the user.
```

Comment:
What filters does the coordinator apply?

Reply:
@abhinavdangeti Do we have a list of specific filters that we can mention here? We took this information from the "https://docs.couchbase.com/server/7.0/learn/services-and-indexes/services/search-service.html#search-service-architecture" I guess, 'filters' is used as a generic term that indicates whatever filters are selected in the index. Let me know if that's correct?

Reply:
I am not aware of any filters, so I don't know what the documentation is talking about.

Reply:
Update: Delete line 51